src/services/recommendationsService.js

```diff
@@ -1,59 +0,0 @@
-import * as recommendationsRepository from '../repositories/recommendationsRepository.js';
-
-const createRecommendation = async ({ name, youtubeLink }) => {
-  const nameExists = await recommendationsRepository.getRecommendationByName(
-    name
-  );
-  const linkExists = await recommendationsRepository.getRecommendationByLink(
-    youtubeLink
-  );
-
-  if (nameExists || linkExists) {
-    return false;
-  }
-
-  await recommendationsRepository.createRecommendation({ name, youtubeLink });
-
-  return true;
-};
-
-const voteRecommendation = async ({ id, type }) => {
-  let updated;
-  if (type === 'up') {
-    updated = await recommendationsRepository.upVote(id);
-  } else {
-    updated = await recommendationsRepository.downVote(id);
-    if (updated.score < -5) await recommendationsRepository.deleteRecommendation(id);
-  }
-
-  if (updated) return true;
-  return false;
-};
-
-const getRandom = async () => {
-  if (Math.random() * 10 > 3) {
-    // 70% of the times: recommendation with score > 10
```

Comment:
Não sei se `* 10 > 3` é referente ao comentário desta linha, se for mova pra cima, caso contrário, explique com um comentário de onde vem `* 10 > 3`